integration-cli/docker_cli_version_test.go

```diff
@@ -57,7 +54,5 @@ func testVersionPlatform(c *check.C, platform string) {
 			break
 		}
 	}
-	if !bFound {
-		c.Errorf("Could not find server '%s' in '%s'", expected, out)
-	}
+	c.Assert(bFound, checker.Equals, true, check.Commentf("Could not fing server '%s' in '%s'", expected, out))
```

Comment:
sorry--this all looks good, but we should fix the minor typo while we're here: "Could not fin**g** server" -> "Could not fin**d** server"